crates/dyn-abi/src/parser.rs

```diff
@@ -202,7 +203,11 @@ impl<'a> TryFrom<&'a str> for TupleSpecifier<'a> {
                 _ => {}
             }
         }
-        types.push(value[start..].try_into()?);
+        // handle termina commas in tuples
+        let candidate = value[start..].trim();
```

Comment:
this was a bug. for `(T,)` it would include the blank string, which would fail parsing into a typespecifier